src/configuration/registrations.ts

```diff
@@ -103,6 +104,15 @@ export function registerDependencies(
     container.register(
         'storageAdapter',
         asFunction(() => {
+            if (
+                process.env.LOCAL_STORAGE_PATH &&
+                process.env.LOCAL_STORAGE_HOSTING_URL
+            ) {
+                return new LocalStorageAdapter(
+                    process.env.LOCAL_STORAGE_PATH || '',
+                    new URL(process.env.LOCAL_STORAGE_HOSTING_URL || ''),
+                );
+            }
```

Comment:
I wonder if we can get rid of the `|| ''`, as empty strings are not valid and we are only doing that to please Typescript :D Maybe something like:

```
const localStoragePath = process.env.LOCAL_STORAGE_PATH || ''; // string
const localStorageHostingUrl = process.env.LOCAL_STORAGE_HOSTING_URL || ''

if (localStoragePath && localStorageHostingUrl) {
    ...
}
```

?